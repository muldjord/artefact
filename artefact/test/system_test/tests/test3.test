INPUT="insertmany.xml"
EXPECT="test3_expected.xml"
FILTER="cat"
TITLE="Insert multiple data entries"
