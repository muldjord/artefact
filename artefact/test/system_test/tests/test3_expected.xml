<?xml version='1.0' encoding='utf-8'?>
<results>
  <result id="0" errcode="0" errstring="Data Success" class="status"
          deviceid="" devicetype="" timestamp="">
  </result>
  <result id="1" errcode="0" errstring="Data Success" class="status"
          deviceid="" devicetype="" timestamp="">
  </result>
  <result id="2" errcode="0" errstring="Data Success" class="status"
          deviceid="" devicetype="" timestamp="">
  </result>
</results>
